lib/assets/i18n/log/messages_en.json

```diff
@@ -177,6 +166,11 @@
   "ZWED0205I":"%s: User logout for auth handler %s",
   "ZWED0206I":"Writing: %s, Size=%s",
   "ZWED0207I":"Wrote: %s",
+  "ZWED0210I":"UNUSED",
+  "ZWED0211I":"RESERVED: The number of processors is: %d",
+  "ZWED0212I":"RESERVED: Environmental variable ZLUX_MIN_WORKERS was not a valid number therefore %s will be used as the minimum workers",
```

Comment:
I feel like ZLUX_MIN_WORKERS and ZLUX_MAX_WORKERS should be references to variable names here
(for example: in case they change... etc)

Reply:
If they ever change it constitutes a breaking API change, so they shouldn't thankfully.